cli/command/container/utils.go

```diff
@@ -92,6 +94,7 @@ func waitExitOrRemoved(ctx context.Context, dockerCli *command.DockerCli, contai
 			}
 		}
 	}()
+	<-started
```

Comment:
I get the change in `client/events.go` but not sure I understand the change in this file. In any case, this seems best effort as it doesn't guarantee any particular execution order.